doc/rules/alias/pow_to_exponentiation.rst

```diff
@@ -0,0 +1,22 @@
+Rule ``pow_to_exponentiation``
+==============================
+
+Converts ``pow()`` to the ``**`` operator.
+
+.. warning:: Using this rule is risky.
+   
+   Risky when the function ``pow()`` is overridden.
+
+Samples
+-------
+
+Example #1.
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+   @@ -1,2 +1,2 @@
+    <?php
+   - pow($a, 1);
+   + $a** 1;
```

Comment:
Minor: space

Reply:
This diff is generated using the actual `pow_to_exponentiation` rule, which means the corresponding fixer does not add the space. This is already covered by the `binary_operator_spaces` rule anyway so nothing to do here IMO.